### ISSUE TITLE
Jazzy ignores NS_OPTIONS?

### ISSUE THREAD

**Issue by colincornaby** [realm/jazzy#538]
Jazzy seems to be documenting my NS_ENUMs, but it's not documenting my NS_OPTIONS. Is this a known issue or am I missing something?

**Comment by jpsim** [comment]
Seems to work fine for me, although jazzy treats it as an enum:

```objc
/// Some options.
typedef NS_OPTIONS(NSUInteger, SomeOptions) {
    /// Foo.
    SomeOptionsFoo = 0,
    /// Bar.
    SomeOptionsBar = 1 << 0,
};
```

![image](https://cloud.githubusercontent.com/assets/474794/14733252/df2ac0ac-0861-11e6-96e3-1a8e6bbc5a59.png)

Is this what you're seeing?

**Comment by colincornaby** [comment]
No, it seems to get ignored. Let me take a look at the commentary around it and see if something is funny.

**Comment by colincornaby** [comment]
I was missing the space in front of the /// which I didn't realize was required. Thanks! Not a bug.

**Comment by jpsim** [comment]
well, the fact that `NS_OPTIONS` are treated as enums I'd consider a bug, so let's keep this open.

**Comment by istx25** [comment]
Any updates on this?

**Comment by jpsim** [comment]
wanna get chummy with libclang?
